docroot/themes/custom/twentyeighteen/sass/components/call-to-action/_call-to-action.scss

```diff
@@ -0,0 +1,34 @@
+// Call to Action Block
+//
+// Styling for three call to action blocks on bottom of homepage
+//
+// Markup: call-to-action.twig
+//
+// Style guide: components.call_to_action
+
+.call-to-action-block {
+  @include span(4 of 12);
+
+  /*  position: relative; */
+}
+
+.call-to-action-text {
+  /* position: absolute; */
+  margin-top: -30%;
+  margin-left: 3%;
+}
+
+.call-to-action-text h1 {
+  font-family: $body_font;
+  color: rgba(254, 218, 0, 1);
```

Comment:
We should have colors defined in variables, but we can fix this after mine gets merged in (you will see I've inserted a bunch of the colors in the _config.scss file once my PR gets merged in.